Source/UniformGradient.swift

```diff
@@ -2,12 +2,19 @@ import UIKit
 
 public struct UniformGradient: Gradient {
 
+    public var angle: GradientAngle?
     public let colors: [UIColor]
+    public let endPoint: CGPoint
     public let startPoint: CGPoint
 
-    public init(colors: [UIColor], startPoint: CGPoint = CGPoint(x: 0.5, y: 0.0)) {
+    public init(colors: [UIColor], startPoint: CGPoint = CGPoint(x: 0.5, y: 0.0), endPoint: CGPoint = CGPoint(x: 0.5, y: 1.0)) {
```

Comment:
_If_ we make the change of removing `startPoint` and `endPoint`, then we won't need this initializer and can remove it entirely.